src/CodeViewer/index.tsx

```diff
@@ -29,7 +42,11 @@ const CodeViewer: React.FunctionComponent<ICodeViewerProps> = ({
 }) => {
   const lang = (language && languageMaps[language]) || language || '';
 
-  const code = React.useMemo(() => highlightCode(value, lang, showLineNumbers), [value, lang, showLineNumbers]);
+  const code = React.useMemo(() => sanitizer.sanitize(highlightCode(value, lang, showLineNumbers), sanitizeConfig), [
+    value,
+    lang,
+    showLineNumbers,
+  ]);
```

Comment:
As far as I understand `highlightCode` returns a HTML string that contains the formatting. Just make sure we don't strip the highlighting. This is a more complicated issue then the one in markdown-viewer.

Reply:
Yup. But the highlighted code only inserts `<span>`s. So I just set `ALLOWED_TAGS: ['span']` and that seems to work:

![image](https://user-images.githubusercontent.com/587740/83214940-c5abdc00-a133-11ea-9f89-10f88e6a1c9a.png)